implementations/micrometer-registry-graphite/src/main/java/io/micrometer/graphite/GraphiteDimensionalNamingConvention.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2020 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.graphite;
+
+import com.google.common.base.Strings;
```

Comment:
Need to strike this unused import and a couple other little checkstyle problems.

Reply:
2e08e9c5e467524292e6faec6df35a7dd9e12c08